azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -495,6 +512,7 @@ def __init__(self, kcsb: Union[KustoConnectionStringBuilder, str]):
         self._mgmt_endpoint = "{0}/v1/rest/mgmt".format(kusto_cluster)
         self._query_endpoint = "{0}/v2/rest/query".format(kusto_cluster)
         self._streaming_ingest_endpoint = "{0}/v1/rest/ingest/".format(kusto_cluster)
+        # TODO: this is somewhat confusing

```

Comment:
Orphan todo comment.
If you intend to handle it in another PR, add ownership i.e. (date, name, plann)